From 3b4993403d04ae7fb85efcc8308c01b22a63861b Mon Sep 17 00:00:00 2001
From: Matthew Barnes <mbarnes@redhat.com>
Date: Thu, 24 Nov 2011 23:01:11 +0000
Subject: Remove all *_DISABLE_DEPRECATED_FLAGS.

I'll care about deprecated APIs again when a gtk4 release date is
announced.  Meanwhile it's just causing distros unnecessary pain.

(cherry picked from commit a28a512c9f839f8ccefeaf4256eb897ce9d6b8f6)
---
diff --git a/configure.ac b/configure.ac
index 4f1edcc..9bd97db 100644
--- a/configure.ac
+++ b/configure.ac
@@ -146,9 +146,7 @@ dnl ******************************
 dnl Compiler Warning Flags
 dnl ******************************
 AS_COMPILER_FLAGS(WARNING_FLAGS,
-	"-DPANGO_DISABLE_DEPRECATED
-	-DGDK_PIXBUF_DISABLE_DEPRECATED
-	-DG_DISABLE_SINGLE_INCLUDES
+	"-DG_DISABLE_SINGLE_INCLUDES
 	-DGTK_DISABLE_SINGLE_INCLUDES
 	-DGSEAL_ENABLE
 	-Wall -Wextra
--
cgit v0.9.0.2
